src/startup.rs

```diff
@@ -76,5 +79,6 @@ fn status_list_routes(state: AppState) -> Router<AppState> {
 
     Router::new()
         .merge(protected_routes)
+        .route("/aggregation", get(aggregation))
```

Comment:
The aggregation endpoint is `/statuslists`
```suggestion
        .route("/", get(aggregation))
```